README.md

```diff
@@ -72,6 +73,7 @@ export RESOURCE_GROUP_NAME=give_your_name_citus_test_automation_r_g
 # export REGION=eastus2
 
 # open and modify the instance types/discs as you wish
+# for valgrind tests, set `numberOfWorkers` to `0` as we do not allocate seperate instances for workers.
```

Comment:
for valgrind tests, set `numberOfWorkers` to `0` as we do not allocate seperate instances for workers.

->

for valgrind tests, set `numberOfWorkers` to `0` as we only use the coordinator.

Reply:
1. 
> Did a quick review, one main thing is the usage of community/enterprise. We have functions that set community or enterprise in the config. Why dont we use them?

For some tests, we call directly run command as I see in README. Other than this, I wanted `run.valgrind` task to be aware of the path we are working on in order to move the logs from there. Hence, I parameterized `run.valgrind` so it also specifies repository. As I already parameterized it, using `use.citus/enterprise` may result in conflicts and seems redundant to me. This why I utilize `use.citus/enterprise` in `run.valgrind`.

2. 

> for valgrind tests, set numberOfWorkers to 0 as we do not allocate seperate instances for workers.
> ->
> for valgrind tests, set numberOfWorkers to 0 as we only use the coordinator.

That makes sense

Reply:
yeah that seems better :)